src/components/tiles/map-tile/floorplan.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+import clsx from 'clsx';
+import { useState } from 'react';
+import { MdLocationPin } from 'react-icons/md';
+
+import { Map } from './map';
+
+const map = [
+  'Regisztrációs pult',
+  'Bosch',
+  'Nokia',
+  'ICF Tech',
+  'Sicontact',
+  'Silicon Labs',
+  'Interactive Brokers',
+  'Paks II.',
+  'Barré',
+  'Nova',
+  'KUKA',
+  'OTP',
+  'Knorr-Bremse',
+  'SCHIS',
+  'Paripa',
+  'BME Suborbitals',
+  'BME Formula Racing Team',
+  'Schdesign',
+  'Kir-dev',
```

Comment:
schdesign
Kir-Dev

I know, the PDF I sen you was also incorrect, but that's not public at least :D